src/Autosuggest.js

```diff
@@ -231,7 +232,24 @@ class Autosuggest extends Component {
             }
             event.preventDefault();
             break;
-
+          case 'Tab': {
+            if (tabToSelect) {
+              event.preventDefault();
```

Comment:
This may need to be moved inside the if below. Users may want to tab to another field when there's no focused suggestion.